example/lib/navigation/navigator_bloc.dart

```diff
@@ -0,0 +1,26 @@
+import 'dart:async';
+import 'package:blemulator_example/navigation/route_name.dart';
+import 'package:bloc/bloc.dart';
+import 'package:flutter/material.dart';
+import './bloc.dart';
+
+class NavigatorBloc extends Bloc<NavigationEvent, dynamic> {
+  final GlobalKey<NavigatorState> navigatorKey;
+
+  NavigatorBloc({this.navigatorKey});
```

Comment:
navigatorKey seems to be not optional. If you want to keep being able to pass it by name, use @required.

Reply:
Indeed, `navigatorKey` is not optional here. Added **@required** annotation.